components/menu/index.css

```diff
@@ -528,8 +540,6 @@ governing permissions and limitations under the License.
 
 .spectrum-Menu-itemLabel {
   grid-area: labelArea;
-  display: flex;
-  align-items: flex-start;
```

Comment:
Flex was causing `  text-overflow: ellipsis;` to not work, was able to refactor the itemLabel to not need to rely on flex. This is what appeared to of broken the truncating functionality.